PowershellTools.Common/Constants.cs

```diff
@@ -16,8 +16,8 @@ internal static class Constants
 
         public const string ReadyEventPrefix = "VsPowershellToolProcess:";
 
-        // 10 seconds
-        public const int HostProcessStartupTimeout = 10000;
+        // 20 minutes
+        public const int HostProcessStartupTimeout = 20 * 60 * 1000; // wait for 20 minutes max for remote powershell host process to initialize

```

Comment:
Eric and I talked yesterday about this. We thought we still need a timeout in case some other exceptions occur and we will indefinitely wait here. With this timeout, we can pop a dialog or something else telling users we have issues and provide a RETRY button for them to reload again. Of course, we don't have this implemented for now. This is more like a placeholder :)